dali/test/python/test_operator_index_reader.py

```diff
@@ -83,4 +84,20 @@ def define_graph(self):
         out_ref = pipe_org.run()
         for a, b in zip(out, out_ref):
             assert np.array_equal(a.as_array(), b.as_array())
-        _ = pipe_org.run()
\ No newline at end of file
+        _ = pipe_org.run()
+
+def test_wrong_feature_shape():
+    features = {
+        'image/encoded': tfrec.FixedLenFeature((), tfrec.string, ""),
+        'image/object/bbox': tfrec.FixedLenFeature([], tfrec.float32, -1.0),
+        'image/object/class/label': tfrec.FixedLenFeature([], tfrec.int64, -1),
```

Comment:
Can you add a comment explaining why an error is expected?

Reply:
Done